lib/event_bridge.rb

```diff
@@ -0,0 +1,30 @@
+require 'aws-sdk-eventbridge'
+
+class EventBridge
+
+  def self.put_event(event_name, payload, prefix, event_bridge)
+    client = Aws::EventBridge::Client.new(
```

Comment:
@PeculiarGoat just leaving this as a note - client can be cached

Reply:
Good point.  Shall I make a card?... I reckon this note will be forgotten here